app/src/main/java/org/mozilla/fenix/components/appstate/AppStoreReducer.kt

```diff
@@ -44,12 +42,7 @@ internal object AppStoreReducer {
             topSites = action.topSites,
             recentBookmarks = action.recentBookmarks,
             recentTabs = action.recentTabs,
-            recentHistory = if (action.recentHistory.isNotEmpty() && action.recentTabs.isNotEmpty()) {
-                val recentSearchGroup = action.recentTabs.find { it is RecentTab.SearchGroup } as RecentTab.SearchGroup?
-                action.recentHistory.filterOut(recentSearchGroup?.searchTerm)
-            } else {
-                action.recentHistory
-            }
```

Comment:
don't worry, it's confusing to me too lol. `recent tabs` is anything to do with `jump back in` and tracking the tabs to show in that content section on the homepage. `Recently visited tabs` is anything to do with the history and search history content section on the homepage. 

so this edit here is just removing the filter, since groups in `recent tabs` will no longer be a thing, so we don't need to account for it in history. that's my understanding of it, anyway.